README.md

```diff
@@ -25,13 +25,29 @@ This action works in conjunction with [im-open/code-coverage-report-generator].
+- In the body of a PR comment this action generates
+- Next to the name of one of the status checks under the `Checks` section of a PR
+- Next to the name of one of the status checks under the `Jobs` section of the workflow run
+- In the body of a status check listed on the workflow run
+
+If thresholds for line or branch coverage have been provided and the actual branch or line coverage does not meet or exceed the threshold, the status check will be marked as `failed`.  Having the status check marked as `failed` will prevent PRs from being merged.  If this status check behavior is not desired, the `ignore-threshold-failures` input can be set and the outcome will be marked as `neutral` if threshold failures are detected.  The status badge that is shown in the comment or status check body will still indicate it was a failure though.
+
+If you want the code coverage to be reported without indicating whether it was a success or failure, leave the `line-threshold` and `branch-threshold` inputs as the default `0`.
+
+There are several factors will contribute to the final disposition:
+
+| Scenario:                                      | `ignore-threshold-`<br/>`failures` input | status check<br/>conclusion | `coverage-outcome`<br/>output | PR Comment or<br/>Status Check<br/>Results Badge |
+|------------------------------------------------|------------------------------------------|-----------------------------|-------------------------------|--------------------------------------------------|
+| `line-threshold:0`<br/>or `branch-threshold:0` | `true` or `false`                        | `neutral`                   | `Passed`                      | N/A                                              |
+| actualCoverage < threshold                     | `false`                                  | `failure`                   | `Failed`                      | `FAILED`                                         |
+| actualCoverage < threshold                     | `true`                                   | `neutral`                   | `Passed`                      | `FAILED`                                         |
+| actualCoverage >= threshold                    | `true` or `false`                        | `success`                   | `Passed`                      | `PASSED`                                         |
 
 ## Limitations
 
-GitHub does have a size limitation of 65535 characters for a Status Check body or a PR Comment.  This action will fail if the contents of the summary file exceed the GitHub [limit].
+GitHub does have a size limitation of 65535 characters for a Status Check body or a PR Comment. This action would fail if the test results exceeded the GitHub [limit]. To mitigate this size issue only details for failed tests are included in the output in addition to a badge, duration info and outcome info. If the comment still exceeds that size, it will be truncated with a note to see the remaining output in the log.
```

Comment:
```suggestion
GitHub does have a size limitation of 65535 characters for a Status Check body or a PR Comment. This action would fail if the test results exceeded the GitHub [limit]. To mitigate this size issue, only details for failed tests are included in the output in addition to a badge, duration info and outcome info. If the comment still exceeds that size, it will be truncated with a note to see the remaining output in the log.
```